tests/src/test/scala/whisk/core/containerpool/docker/test/DockerContainerFactoryTests.scala

```diff
@@ -57,6 +56,8 @@ class DockerContainerFactoryTests
 
   behavior of "DockerContainerFactory"
 
+  val byteSize = ByteSize.fromString("1024 m")
```

Comment:
Is there a special reason for parsing a String instead of using `1024.MB` directly?

Reply:
Oh, no special reason.
Already changed as below, it seems more make scence.
```
val defaultUserMemory: ByteSize = 1024.MB
```